templates/cluster_config_ami_builder.json.erb

```diff
@@ -152,7 +155,8 @@
         "instances": {
           "number_of_instances": 0,
           "instance_type": "<%= admin_instance_type %>",
-          "root_device_type": "ebs"
+          "root_device_type": "ebs",
+          "root_device_size": 16
```

Comment:
That's an option but I'm not sure I see the value in that. I see this is a useful but not incredibly important feature, and in some ways it's an anti-pattern because we *should not* store state or logs we care about on ebs volumes.